pkg/backingstore/backingstore.go

```diff
@@ -350,9 +350,17 @@ func createCommon(cmd *cobra.Command, args []string, storeType nbv1.StoreType, p
 	o = util.KubeObject(bundle.File_deploy_internal_secret_empty_yaml)
 	secret := o.(*corev1.Secret)
 	secret.Namespace = options.Namespace
-	secret.Name = fmt.Sprintf("backing-store-%s-%s", storeType, name)
-	secret.StringData = map[string]string{}
-	secret.Data = nil
+	if secretName != "" {
+		secret.Name = secretName
+		if !util.KubeCheck(secret) {
+			log.Fatalf(`❌ Could not find the suggested secret: name %q namespace %q`, secret.Name, secret.Namespace)
+			return
+		}
+	} else {
```

Comment:
`createCommon` is also running in functions such as `RunCreatePVPool`.
Do we actually need a secret in all the functions that call it?
Do we actually want to fail if there is no secret? 
@jackyalbo @baum  WDYT?

Reply:
@liranmauda, the "secret-name" flag is defined for the following commands only:
- [RunCreateAWSS3](https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L109-L112)
- [RunCreateS3Compatible](https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L139-L142)
- [RunCreateIBMCos](https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L173-L176)
- [RunCreateAzureBlob](https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L203-L206)
- [RunCreateGoogleCloudStorage](https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L225-L228)

And it is not defined as a valid flag for the RunCreatePVPool,
see:
- https://github.com/noobaa/noobaa-operator/blob/24022e21a3881fb64d2ddf1eccf799eb4c6c4f01/pkg/backingstore/backingstore.go#L241-L276

Reply:
I see 😄

Reply:
There is a secret for pv-pool where we save the connection string, but not fully sure if it make sense to get it from the costumer. maybe we just need to make sure we don't get secretName when in PVPool (didn't check if we already do) maybe @romayalon knows better as she added the original feature.

Reply:
According to the snippet @baum added, we cannot get a secret from the customer when we are in PVPool:

```
$ noobaa backingstore create pv-pool test --secret-name aaa
Error: unknown flag: --secret-name
```

It seems that it doesn't make sense (As @jackyalbo wrote) to get the connection string from the user (Well, at list that I can currently think of).
@romayalon to keep us honest.